roles/StackStorm.st2repo/tasks/st2repo_dnf.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Fixes "Failure talking to yum: Cannot retrieve repository metadata (repomd.xml) for repository: StackStorm_stable. Please verify its path and try again" when installing st2
+- name: Update ca-certificates package
+  become: yes
+  yum:
+    name: ca-certificates
+    state: latest
+  register: _task
+  retries: 5
+  delay: 3
+  until: _task is succeeded
+  tags: [st2repo, skip_ansible_lint]
+
+# See: https://github.com/docker-library/docs/tree/master/centos#package-documentation
+# We ship `nginx.conf` via `st2` package doc files, for example
+- name: Enable shipping package documentation files for EL
+  become: yes
+  ini_file:
+    dest: /etc/yum.conf
+    section: main
+    option: tsflags
+    value: nodocs
+    state: absent
+  when: ansible_facts.os_family == "RedHat"
+  tags: st2repo
+
+- name: Add StackStorm repo
+  become: yes
+  yum_repository:
```

Comment:
Yeah, indeed many `_dnf.yml` files are identical to `_yum.yml` files.
We definitely want to avoid code duplication and use one set of rules to manage both.

Reply:
I agree with @armab's suggestion